jina/flow/__init__.py

```diff
@@ -790,6 +789,83 @@ def my_reader():
         """
         self._get_client(**kwargs).search(input_fn, output_fn, **kwargs)
 
+    def plot(self, output: str=None, copy_flow: bool = False) -> 'Flow':
+        """
+        Creates a mermaid graph for the Flow.
+        If a file name is provided it will create a jpg image with that name,
+        otherwise it will display the URL for mermaid.
+        If called within IPython notebook, it will be rendered inline,
+        otherwise an image will be created.
+
+        Example,
+
+        .. highlight:: python
+        .. code-block:: python
+
+            flow = Flow().add(name='pod_a').plot('flow_test.jpg')
+
+        :param output: a filename specifying the name of the image to be created
+        :return: the flow
+        """
+
+        op_flow = copy.deepcopy(self) if copy_flow else self
+        url = ''
+
+        mermaid_graph = ['graph TD']
+        for node, v in self._pod_nodes.items():
+            for need in sorted(v.needs):
+                mermaid_graph.append(f'{need}[{need}] --> {node}[{node}]')
+        mermaid_str = '\n'.join(mermaid_graph)
+        if output:
+            self._mermaidstr_to_jpg(mermaid_str, output)
+            try:
+                from IPython.display import Image
+                from IPython.display import display
+                pil_img = Image(filename=output)
+                display(pil_img)
+            except ModuleNotFoundError:
+                self.logger.error('Failed to import IPython')  
```

Comment:
```suggestion
                self.logger.debug('Failed to import IPython')  
```

no need to panic the user for this